### DISCUSSION TITLE
Inherit configs in pydantic v2

### DISCUSSION THREAD

**Discussion by ilija-lazoroski** [pydantic/pydantic#7778] [Question]
Hey,

In version 1 we could do the following:

```python
class ImutableConfig:
    class Config:
        allow_mutation = False
        extras = Extra.forbid

class MutableConfig(ImutableConfig):
   class Config:
      allow_mutation = True
```

I don't really see a way to do a similar thing using `model_config: ConfigDict.`

**Comment by lelecanfora** [comment]
I have the exact same problem with a very banal issue, how can I change the "title" value in the config:


```python
from uuid import UUID

from pydantic import BaseModel, ConfigDict

class Base(BaseModel):
    model_config = ConfigDict(
        from_attributes=True,
        populate_by_name=True,
        title='parent'
    )

class BaseId(Base):
    id: UUID

class Child(BaseId):
    pass
```
how can I give a custom title to the class Child?

**Reply by rusmux** [reply]
Model configs are merged: https://docs.pydantic.dev/2.7/concepts/config/#change-behaviour-globally
